model/item.h

```diff
@@ -70,16 +70,16 @@ namespace minsky
       // extend each item by a portradius to solve ticket #903
       return left-portRadius<=x && right+portRadius>=x && bottom+portRadius>=y && top-portRadius<=y;
     }
-    bool valid() const {return left!=right;}
+    bool valid() const {return right-left!=2*portRadius;}
```

Comment:
What's the reasoning here? On initialisation, left=right=0, so with your change, initialliy the bounding box is valid. That doesn't seem right.

Reply:
True. I became confused because of the addition of the portRadius on all sides of the bounding box. Reverting it to its previous definition.